src/main/java/org/jabref/logic/ai/chatting/model/JvmOpenAiChatLanguageModel.java

```diff
@@ -59,6 +57,7 @@ public Response<AiMessage> generate(List<ChatMessage> list) {
                 .model(aiPreferences.getSelectedChatModel())
                 .temperature(aiPreferences.getTemperature())
                 .n(1)
+                .maxTokens(2048)
```

Comment:
Why this? @InAnYan needs to carefully check!

Reply:
Thanks for your guidance, I have fixed the code you mentioned.